src/datasource.ts

```diff
@@ -98,6 +98,16 @@ export class DataFrameDataSource extends DataSourceApi<DataframeQuery> {
     );
   }
 
+  async queryTables(query: string) {
+    var filter = `name.Contains("${query}")`;
+
+    return lastValueFrom(
```

Comment:
Could you explain your use of `lastValueFrom` and `take: 5` here? Do we really only want to show 5 options?

Reply:
`lastValueFrom` converts the observable to a promise. Since this isn't a stream of data, promises are easier to work with (in my opinion at least).

I chose a take of a 5 because it's a reasonable amount to fit on the screen at one time:

<img width="334" alt="Screen Shot 2022-10-20 at 5 28 20 PM" src="https://user-images.githubusercontent.com/9257800/197071020-e5ac6add-8fbb-4b45-9c56-161ba7a9943b.png">